djangocms_versioning/operations.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+import uuid
+
+from .models import Version
+from .signals import post_version_operation, pre_version_operation
+
+
+def send_pre_version_operation(operation, sender=Version, **kwargs):
+    token = str(uuid.uuid4())
```

Comment:
@monikasulik Think this is similar to how CMS signals work, where you can tie pre/post signals using that token

Reply:
Well, I'm not familiar with how they used to work and since from 4.0 they won't be there, I still think this should be documented and explained :) Not even sure what "tying" a signal means.